### PR TITLE
Added the required code

### PR DESCRIPTION
**Pull request by srisatya1197**
@rcoder23  I added the required code for the #340 . Please review it and let me know any changes to be made.

### CODE REVIEW

**Review by rcoder23 — CHANGES_REQUESTED** [review]
Add driver code also .

**Comment by srisatya1197** [comment]
@rcoder23 Can you please check now.

File: Topic Wise Questions with Solutions/Linked List/remove.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+
+"""
+nums=list(map(int,input().split()))
+val=int(input())
+a=nums.count(val)
+i=1
+while i <=a:
+nums.remove(val)
+i+=1
```
Comment (rcoder23) [inline Topic Wise Questions with Solutions/Linked List/remove.py:23]: indention error

File: Topic Wise Questions with Solutions/Linked List/remove.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+Constraints:
+
+0 <= nums.length <= 100
+0 <= nums[i] <= 50
+0 <= val <= 100
+
+
+"""
+nums=list(map(int,input().split()))
+val=int(input())
```
Comment (rcoder23) [inline Topic Wise Questions with Solutions/Linked List/remove.py:18]: Take input with message

**Review by rcoder23 — CHANGES_REQUESTED** [review]
there is no connection Between driver code and solution code.

**Comment by srisatya1197** [comment]
@rcoder23 Sorry for the delay, I had some exams. Can you please check now I changed the code.

**Comment by srisatya1197** [comment]
@rcoder23 Can you please check this.

**Comment by srisatya1197** [comment]
@rcoder23 Can you please check?

File: Topic Wise Questions with Solutions/Linked List/remove.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+Constraints:
+
+0 <= nums.length <= 100
+0 <= nums[i] <= 50
+0 <= val <= 100
+
+
+"""
+print("Enter the values for nums to store in an array:")
```
Comment (rcoder23) [inline Topic Wise Questions with Solutions/Linked List/remove.py:17]: **Enter the values for nums to store in an array:**
change to **Enter the values for nums to store in an linkedist**

File: Topic Wise Questions with Solutions/Linked List/remove.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+nums=list(map(int,input().split()))
+print("Enter the value of val:")
```
Comment (rcoder23) [inline Topic Wise Questions with Solutions/Linked List/remove.py:19]: **Enter the value of val to delete :**

**Review by rcoder23 — CHANGES_REQUESTED** [review]

**Comment by srisatya1197** [comment]
@rcoder23 Can you please check now ?

**Comment by srisatya1197** [comment]
@rcoder23 Can you please check now?

**Comment by srisatya1197** [comment]
@rcoder23 Can you please check this pr.

**Review by rcoder23 — APPROVED** [review]

**Comment by srisatya1197** [comment]
Is it merged @rcoder23 ?

**Comment by rcoder23** [comment]
@srisatya1197 Sorry I forget to merged.

**Comment by srisatya1197** [comment]
@rcoder23 thank you.

**Pull request merged**